src/libopenrave/kinbody.cpp

```diff
@@ -4298,15 +4298,15 @@ void KinBody::_ComputeInternalInformation()
     if((_veclinks.size() > 0)&&(_vecjoints.size() > 0)) {
         std::vector< std::vector<int> > vlinkadjacency(_veclinks.size());
         // joints with only one attachment are attached to a static link, which is attached to link 0
-        for( const auto joint :_vecjoints) {
```

Comment:
@lazydroid can you use `const JointPtr&` instead? Thanks

Reply:
pipeline #479997